
--- a/src/r8125_rss.c
+++ b/src/r8125_rss.c
@@ -88,7 +88,7 @@ int rtl8125_get_rxnfc(struct net_device *dev, struct ethtool_rxnfc *cmd,
         struct rtl8125_private *tp = netdev_priv(dev);
         int ret = -EOPNOTSUPP;
 
-        netif_info(tp, drv, tp->dev, "rss get rxnfc\n");
+        //netif_info(tp, drv, tp->dev, "rss get rxnfc\n");
 
         if (!(dev->features & NETIF_F_RXHASH))
                 return ret;
@@ -153,7 +153,7 @@ static int rtl8125_set_rss_hash_opt(struct rtl8125_private *tp,
 {
         u32 rss_flags = tp->rss_flags;
 
-        netif_info(tp, drv, tp->dev, "rss set hash\n");
+        //netif_info(tp, drv, tp->dev, "rss set hash\n");
 
         /*
          * RSS does not support anything other than hashing
@@ -261,7 +261,7 @@ int rtl8125_set_rxnfc(struct net_device *dev, struct ethtool_rxnfc *cmd)
         struct rtl8125_private *tp = netdev_priv(dev);
         int ret = -EOPNOTSUPP;
 
-        netif_info(tp, drv, tp->dev, "rss set rxnfc\n");
+        //netif_info(tp, drv, tp->dev, "rss set rxnfc\n");
 
         if (!(dev->features & NETIF_F_RXHASH))
                 return ret;
@@ -286,7 +286,7 @@ u32 rtl8125_get_rxfh_key_size(struct net_device *dev)
 {
         struct rtl8125_private *tp = netdev_priv(dev);
 
-        netif_info(tp, drv, tp->dev, "rss get key size\n");
+        //netif_info(tp, drv, tp->dev, "rss get key size\n");
 
         if (!(dev->features & NETIF_F_RXHASH))
                 return 0;
@@ -298,7 +298,7 @@ u32 rtl8125_rss_indir_size(struct net_device *dev)
 {
         struct rtl8125_private *tp = netdev_priv(dev);
 
-        netif_info(tp, drv, tp->dev, "rss get indir tbl size\n");
+        //netif_info(tp, drv, tp->dev, "rss get indir tbl size\n");
 
         if (!(dev->features & NETIF_F_RXHASH))
                 return 0;
@@ -319,7 +319,7 @@ int rtl8125_get_rxfh(struct net_device *dev, u32 *indir, u8 *key,
 {
         struct rtl8125_private *tp = netdev_priv(dev);
 
-        netif_info(tp, drv, tp->dev, "rss get rxfh\n");
+        //netif_info(tp, drv, tp->dev, "rss get rxfh\n");
 
         if (!(dev->features & NETIF_F_RXHASH))
                 return -EOPNOTSUPP;
@@ -383,7 +383,7 @@ int rtl8125_set_rxfh(struct net_device *dev, const u32 *indir,
         int i;
         u32 reta_entries = rtl8125_rss_indir_tbl_entries(tp);
 
-        netif_info(tp, drv, tp->dev, "rss set rxfh\n");
+        //netif_info(tp, drv, tp->dev, "rss set rxfh\n");
 
         /* We require at least one supported parameter to be changed and no
          * change in any of the unsupported parameters
